int32[] occGrid
int32[] occGrid_shape
int32[] warped
int32[] warped_shape
int32[] x0
int32[] x1
int32[] x2
int32[] y0
int32[] y1
int32[] y2